bioio/writers/__init__.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
 
-from .ome_tiff_writer import OmeTiffWriter
-from .ome_zarr_writer import OmeZarrWriter
-from .ome_zarr_writer_2 import OmeZarrWriter as OmeZarrWriter2
+from importlib.metadata import entry_points as _entry_points
+from typing import List
 
-__all__ = [
-    "OmeTiffWriter",
-    "OmeZarrWriter",
-    "OmeZarrWriter2",
-]
+# Public API list
+__all__: List[str] = []
+
+# Discover all registered writers
+_eps = _entry_points(group="bioio.writers")
```

Comment:
how do writers get registered?  This seems to need some documentation somewhere maybe?

Reply:
I'll open another documentation branch once the writers are on main for their respective plugins. That way we can reference directly.